bundles/ranvier-commands/commands/get.js

```diff
@@ -63,6 +63,7 @@ module.exports = (srcPath) => {
       player.addItem(item);
 
       Broadcast.sayAt(player, `<green>You receive loot: </green>${item.display}<green>.</green>`);
+      Broadcast.sayAt(room, `<green>${player.name} receives loot: </green>${item.display}<green>.</green>`)
```

Comment:
The default functionality is just "see item in room, get foo, oh it's not there anymore, oh well" That's a much more preferable scenario to adding in a ton of code to debounce events to players to prevent abuse that can just be avoided